sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/DefaultOfferRequirementProvider.java

```diff
@@ -338,6 +342,13 @@ private static String getConfigTemplateDownloadPath(ConfigFileSpec config) {
         return String.format("%s%s", CONFIG_TEMPLATE_DOWNLOAD_PATH, config.getName());
     }
 
+    private static Protos.DiscoveryInfo getDiscoveryInfo(String dnsName) {
+        return Protos.DiscoveryInfo.newBuilder()
+                .setVisibility(Protos.DiscoveryInfo.Visibility.CLUSTER)
```

Comment:
This is fine.  I wonder whether we'd ever want to expose this as configurable.  This would change the YAML format right?